surrealdb/clients/ws.py

```diff
@@ -82,14 +100,13 @@ async def connect(self) -> None:
 
         if self._token is not None:
             await self.authenticate(self._token)
-
         if self._username is not None and self._password is not None:
             await self.signin(username=self._username, password=self._password)
-
```

Comment:
why are we removing these whitespaces? it separates out the logic and removing them is harder to read